Bottom: c43ea98c75ef0f4c7e044b494e71473179c1639a
Top:    cc65e5e4d416dc45be9330f6163eef54529a2d54
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:20 +0000

Reduce tolerances a smidgin


---

diff --git a/src/netclass.c b/src/netclass.c
index 54c6d6e..6003f1b 100644
--- a/src/netclass.c
+++ b/src/netclass.c
@@ -31,16 +31,16 @@ static char *netclass_names[10] =
 
 static Coord clearances[10][10] =
   {
-    {MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3)}
+    {MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3)}
   };
